protocol/x/vault/keeper/vault.go

```diff
@@ -3,23 +3,73 @@ package keeper
 import (
 	"math/big"
 
+	errorsmod "cosmossdk.io/errors"
 	"cosmossdk.io/store/prefix"
-	storetypes "cosmossdk.io/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/dydxprotocol/v4-chain/protocol/lib/log"
 	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 	"github.com/dydxprotocol/v4-chain/protocol/x/vault/types"
 )
 
+// GetMegavaultEquity returns the equity of the megavault (in quote quantums), which consists of
+// - equity of the megavault main subaccount
+// - equity of all non-deactivated vaults
```

Comment:
nit: Let's sync on this, not sure we need to exclude deactivated vaults if they have equity since it should be possible to withdraw from deactivated vaults.

Reply:
synced offline. decided to include deactivated vaults